City-of-Helsinki/drupal-hdbt-admin
# UHF-922 Fixes for diff for images

To test:

1. Run `composer require drupal/hdbt_admin:dev-UHF-922-diff-fixes` on any setup you have.
2. Go to some content that has list of links with pictures.
3. Drag and drop them to different order and change one picture. Save the content.
4. Go to Revisions and compare your latest revisions. The layout should look like the attached screenshot.

<img width="910" alt="Screenshot 2021-03-12 at 15 23 30" src="https://user-images.githubusercontent.com/2276077/110945879-e94cb580-8346-11eb-91b9-c7808da20997.png">
<img width="801" alt="Screenshot 2021-03-12 at 15 23 07" src="https://user-images.githubusercontent.com/2276077/110945834-dc2fc680-8346-11eb-97b2-6ff779bd1342.png">
